.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "recommendations": [
+    "ms-vscode-remote.vscode-remote-extensionpack",
+    "ms-vscode-remote.remote-containers",
+    "ms-azuretools.vscode-docker",
+    "GitHub.vscode-pull-request-github",
+    "ms-vsliveshare.vsliveshare"
```

Comment:
It's an amazing collaboration tool that I cannot recommend strongly enough, but, if I'm being honest may not belong here... If you'd prefer, here's an alternative suggestion.
```suggestion
```